x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -154,14 +148,12 @@ export const ProcessEventDot = styled(
         [dispatch, nodeId]
       );
 
-      const handleClick = useCallback(
-        (clickEvent: React.MouseEvent<SVGSVGElement, MouseEvent>) => {
-          if (clickTargetRef.current !== null) {
-            (clickTargetRef.current as any).beginElement();
-          }
-        },
-        [clickTargetRef]
-      );
+      const handleClick = useCallback(() => {
+        if (animationTarget.current !== null) {
+          // Use undocumented `beginElement` API on SVG element
```

Comment:
@oatkiller it's documented in SVG 1.1 ( Rec status since 16 Aug 2011) https://www.w3.org/TR/SVG11/animate.html#__smil__ElementTimeControl__beginElement

Reply:
I'll add that link to the comment

Reply:
https://github.com/elastic/kibana/pull/61886/commits/d65818852d6772a18a38533b22848ebb6e7f120e @bkimmel